expression/integration_test.go

```diff
@@ -4046,3 +4046,11 @@ func (s *testIntegrationSuite) TestIssue9325(c *C) {
 	result = tk.MustQuery("select * from t where a < timestamp'2019-02-16 14:21:00'")
 	result.Check(testkit.Rows("2019-02-16 14:19:59", "2019-02-16 14:20:01"))
 }
+
+func (s *testIntegrationSuite) TestIssue9710(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	for i := 0; i < 40; i++ {
```

Comment:
This for loop is used to test whether it truncate the floating part?

Reply:
Yes, but this UT is a bit hard to understand and explain...
So I remove it and use a new approach to test it..